src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandler.java

```diff
@@ -105,9 +108,28 @@ private void logSentFor(final Interchange interchange) {
     private void logRecepSentFor(final Interchange interchange) {
         if (LOGGER.isInfoEnabled()) {
             final var header = interchange.getInterchangeHeader();
-            LOGGER.info("Published for async send to MESH a RECEP for the interchange from "
+            LOGGER.info("Published for async send to MESH an NHSACK for the interchange from "
                     + "Sender={} to Recipient={} with RIS={}",
                 header.getSender(), header.getRecipient(), header.getSequenceNumber());
         }
     }
+
+    public static class MessageProcessingResult {
```

Comment:
I guess we don't want to instantiate this directly:
```suggestion
    public static abstract class MessageProcessingResult {
```